### ISSUE TITLE
[PRE REVIEW]: Learning 'Learning Curves' with R Shiny

### ISSUE THREAD

**Issue by whedon** [openjournals/jose-reviews#76]
**Submitting author:** @uriSISlab (<a href="http://orcid.org/0000-0003-4104-8505">Nicholas Bernardo</a>)
**Repository:** <a href="https://github.com/uriSISlab/LearningCurveApp" target ="_blank">https://github.com/uriSISlab/LearningCurveApp</a>
**Version:** v1.0.0
**Editor:** Pending
**Reviewer:** Pending

**Author instructions**

Thanks for submitting your paper to JOSE @uriSISlab. **Currently, there isn't an JOSE editor assigned** to your paper.

@uriSISlab if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/jose-reviewers) have already agreed to review for JOSE and may be suitable for this submission.

**Editor instructions**

The JOSE submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.84  T=0.02 s (323.5 files/s, 60125.3 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
R                                2            156             53            554
Rmd                              2             76            113            212
Markdown                         2             24              0             51
TeX                              1             17              0             45
-------------------------------------------------------------------------------
SUM:                             7            273            166            862
-------------------------------------------------------------------------------


Statistical information for the repository '76' was gathered on 2020/02/09.
No commited files with the specified extensions were found.
```

**Comment by whedon** [comment]
Failed to discover a valid open source license.

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1080/001401300184404 is OK
- 10.1080/00207540110036696 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/jose-papers/blob/jose.00076/jose.00076/10.21105.jose.00076.pdf)

**Comment by nbernardo213** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List Whedon's capabilities
@whedon commands

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

EDITORIAL TASKS

# Compile the paper
@whedon generate pdf

# Compile the paper from alternative branch
@whedon generate pdf from branch custom-branch-name

# Ask Whedon to check the references for missing DOIs
@whedon check references

# Ask Whedon to check repository statistics for the submitted software
@whedon check repository

```

**Comment by nbernardo213** [comment]
@whedon - Thank you for your email. We have looked at the list and we would like to request Andrew Duncan (aj2duncan) to review our submission. Please let us know if you require any additional information.

Best, @uriSISlab

**Comment by whedon** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

```
@whedon commands
```

**Comment by labarba** [comment]
hi @uriSISlab (Nicholas Bernardo) — In an audit of unattended old submissions (we built a backlog during the pandemic), we found your submission from February 2020. My sincere apologies for neglecting to assess your submission, and get back to you! We really dropped the ball here. 

In looking through your repository and reading the paper, I do find that this is a fairly small software project (550 loc) and it would not fit our scope and guidelines for JOSE. I'm going to go ahead and reject this submission now. We want to be mindful that JOSE papers are taken seriously by the community, and are thus looking for a substantial scholarly contribution from each publication. 

Thank you for your understanding, and I wish you the best in your shiny endeavors!
